conf.yaml

```
@@ -1927,7 +1927,7 @@ contents:
             current:    *stackcurrent
             index:      x-pack/functionbeat/docs/index.asciidoc
             branches:   [ master, 8.0, 7.17, 7.16, 7.15, 7.14, 7.13, 7.12, 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5 ]
```

Shouldn't be master is `{main: master}` here as well?
```suggestion
            branches:   [ {main: master}, 8.0, 7.17, 7.16, 7.15, 7.14, 7.13, 7.12, 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5 ]
```